integration-tests/data/extended_multisig.ts

```diff
@@ -0,0 +1,73 @@
+export const extended_multisig = `{ parameter
```

Comment:
The file doesn't seem to be used and could be deleted

Reply:
whoops, from another project. Deleted :)